pytest_tap/plugin.py

```diff
@@ -20,7 +20,7 @@
 
 def pytest_addoption(parser):
     """Include all the command line options."""
-    group = parser.getgroup('terminal reporting', after='general')
+    group = parser.getgroup('terminal reporting', 'reporting', after='general')
```

Comment:
Could you speak to what is going on with this line change (if for nothing more than a bit of historical recording)?